test/source/mock/google/google-data.ts

```diff
@@ -37,16 +97,27 @@ export class GoogleData {
     }
   }
 
-  public storeSentMessage = (parsedMail: ParsedMail): string => {
+  public storeSentMessage = (parsedMail: ParsedMail, base64Msg: string): string => {
+    let attId = '';
+    let signatureAtt: Attachment | undefined;
+    if (parsedMail.attachments?.length) {
+      const att = parsedMail.attachments.find(a => ['encrypted.asc', 'signature.asc'].includes(a.filename!));
+      signatureAtt = parsedMail.attachments.find(a => 'signature.asc' === a.filename);
+      if (att) {
+        attId = Util.lousyRandom();
+        DATA[this.acct].attachments[attId] = { data: att.content.toString('base64'), size: att.size };
+      }
+    }
     const barebonesGmailMsg: GmailMsg = { // todo - could be improved - very barebones
       id: `msg_id_${Util.lousyRandom()}`,
       threadId: null, // tslint:disable-line:no-null-keyword
       historyId: '',
       labelIds: ['SENT' as GmailMsg$labelId],
       payload: {
         headers: [{ name: 'Subject', value: parsedMail.subject }],
-        body: { data: parsedMail.text, attachmentId: '', size: parsedMail.text.length }
+        body: { data: parsedMail.text, attachmentId: attId, size: parsedMail.text?.length }
```

Comment:
This looks suspicious - if `parsedMail.text` can be undefined, that means `data` and `size` can remain undefined which seems weird. Will investigate